### DISCUSSION TITLE
Blocking whatsapp.

### DISCUSSION THREAD

**Discussion by fizsef** [AdguardTeam/AdGuardHome#2881] [Q&A]
I'm trying to block the service and the use of the whatsapp app for the client. Specified in the client settings as a blocked service. But you can still use whatsapp on this device without any problems. Everything you need to do to make it work? 
Version 105.2.
![1](https://user-images.githubusercontent.com/74646605/112643854-d9a89300-8e55-11eb-9a3a-0b95ac2bb2b4.jpg)
![2](https://user-images.githubusercontent.com/74646605/112643858-dad9c000-8e55-11eb-85e0-e16813a0a40f.jpg)

**Comment by ameshkov** [comment]
As I recall, the only way is to make sure that Whatsapp **never** learns the IP addresses, otherwise it uses cached IPs

**Comment by smic717394** [comment]
Hi, I have tje same issue, blocking whatsapp and telegram , wull only block the web version , tje app still works without any problem
